schema/user.js

```diff
@@ -26,4 +39,4 @@ const userResolvers = {
   },
 };
 
-export { typeDef, userResolvers };
```

Comment:
you might need to mp the response from REST API to the schema before returning data as the json format of the schema is different

Reply:
Not sure how to do that , I created the schema thinking we will not use REST api in future, If we are using the existing REST API , then It would be better If we just continue with existing structure.